### ISSUE TITLE
Contact Form Does Not set Reply To Value Correctly

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/pul_library_drupal#1999]
### Expected behavior
When using the website's contact form to distribute mail to libanswers the Reply-To value needs to be set the email address of the individual who submitted the form. 

### Actual behavior
Currently this is what the mail headers look like:

```
To: myqueue@princeton.libanswers.com
From: Form Submitter's Name via Princeton University Library <pulweb@princeton.edu>
Reply-to:  Form Submitter's Name via Princeton University Library <pulweb@princeton.edu>,
         Form Submitter's Name <formsubmittersemail@princeton.edu>
```

### Steps to replicate
Use the https://library.princeton.edu/contact-us form. 

### Impact of this bug
Libanswers ticket managers need to manually change the email of the ticket to that of the submitter. It's very easy to forget to do this. 

### Implementation notes, if any
This behavior started on 4/7/2023. It does not happen with forms on the special collections website running Webform version webform-7.x-4.24. We upgrading to webform-7.x-4.25 (a bugfix release non-security release on 4/7 in production). However rolling back to -7.x-4.24 in this branch does not return the former behavior when tested in staging.

**Comment by maxkadel** [comment]
While the error started in conjunction with a deploy; however, rolling back the changes in that deploy is not fixing the issue.

The system does not log configuration changes, so the issue may be there?

Could possibly hard-code a patch, but not desirable.

**Comment by kevinreiss** [comment]
@sandbergja took a fresh look at this me and we bypassed the issue by changing the general configuration of the webform module at: https://library.princeton.edu/admin/config/content/webform. We removed the default email address and sender name values. This values were being injected and overriding the individual values the individual webform had specified for the sender and reply-to mail header values.
